content/blog/2021/06/2021-06-17-libera-chat.adoc

```diff
@@ -0,0 +1,96 @@
+
+A more detailed history of the transition is available on the link:https://groups.google.com/g/jenkinsci-dev/c/pEcun5ZSfgo/m/ztRg03VNAgAJ[Jenkins developers mailing list].
+
+=== What did we move?
+
+We have created the following IRC chats:
+
+that currently live on freenode to libera.chat:
+
+* `#jenkins` - general discussion channel for Jenkins users and developers.
+  We no longer consider it a central chat for all things Jenkins, but we intend to keep it available to users
+  who want to keep using IRC.
+* `#jenkins-hosting` - Chat used by the link:/project/teams/hosting/[Hosting Team] to manage hosting automation
+* `#jenkins-infra` - Chat used by the link:/projects/infrastructure/#team[Infrastructure Team] to coordinate efforts related to Jenkins project infrastructure and the incident response
+* `#jenkins-release`- Chat used for Jenkins core release coordination by the link:https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#team[Release Team]
+
+All other channels have NOT been moved to Libera Chat IRC and were replaced by other channels.
+Most notable channels:
+
+* `#jenkins-meeting` - the link:/project/governance-meeting/[Jenkins Governance Meeting] uses combination of Zoom and Google Docs at the moment. We do not longer use the IRC chat for that purpose.
+* `#jenkins-cert` - no private chat going forward. The link:/security[Security Team] will use mailing lists only going forward
+* `#jenkins-community` - Replaced by the link:https://gitter.im/jenkinsci/advocacy-and-outreach-sig[Advocacy&Outreach] Gitter channel.
+
+=== Freenode concerns and disclaimers
+
+Going forward, the Jenkins community does not have **ANY** official channels on Freenode.
+All channels are either removed or left abandoned.
+We no longer manage or control our channels.
+Most of the Jenkins IRC channel operators use IRCCloud,
+and hence they link:https://twitter.com/oleg_nenashev/status/1403265278566412288[were banned] by the Freenode team on June 11th.
+Only a few operators have reconnected since this event.
+
+Taking the takeover of Freenode and hostile actions taken by the new management,
+we can no longer guarantee any of the following:
+
+* Authenticity of registered users on Freenode.
+  All IRC accounts, cloaked or not, might have been taken over by the new Freenode management.
+   Do not trust messages from the Jenkins contributor account IDs there.
+* Confidentiality of private channels like `#jenkins-cert`.
+  We performed audit of the conversations there, and we believe there are no undisclosed security vulnerabilities referenced in the channel.
+   After all, this IRC has been rather dormant for the last 18 months.
+ * IRC User Passwords.
+   All passwords on Freenode IRC may be compromised in the future.
+   We strongly advice all users to rotate their passwords if they were used on Freenode IRC.
+   
+ In the coming weeks we will publish backups of IRC channel conversations somewhere in our GitHub repositories.
```

Comment:
Will we? That’s against the free node policies of the time? Not providing a public log unless explicitly stated in the topic